beacon-chain/db/kv/kv.go

```diff
@@ -47,6 +47,10 @@ var (
 		Name: "validator_entry_cache_miss",
 		Help: "The total number of cache misses on the validator entry cache.",
 	})
+	validatorEntryCacheDelete = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "validator_entry_cache_delete",
```

Comment:
this can better conformed to naming standard: https://prometheus.io/docs/practices/naming/

how does `validator_entry_cache_delete_total` sound? Same with the above as well